testing_guide.md

```diff
@@ -0,0 +1,691 @@
+[...]
+# The first syncing peer isn't always the fastest.
+2022-09-18T10:20:24Z [net] Initial headers sync complete with peer=16: releasing all at height=752041 (redownload phase)
+```
+
+**After you finish testing the pre-sync phase, it is encouraged to test further and verify that a new node is able to sync the full blockchain.**
+
+Once the test is successful, don't forget to [stop your node and clean the datadir](#4-reset-testing-environment).
+
+## Testing the GUI
+
+Since you're starting from a fresh datadir, the first thing you should see is the progress screen of Initial Block Download (IBD). You can simply hide it at the bottom right and proceed with testing.
+
+> :information_source: If you are running on signet, the window title should be "Bitcoin Core - [signet]". See [this guide](https://github.com/bitcoin/bitcoin/issues/24501#issuecomment-1081463097) on how to switch to signet from within the GUI.
+
+### Testing your localized GUI
+
+**Are you a non-English native speaker?** Why not explore the GUI while contributing to a better experience for all?
+
+> :information_source: You can test a localization using `-lang=<lang>`.
+
+The GUI offers an easy way to navigate around using `<Alt>+<Key>` shortcuts, where the `<Key>` is part of the option's text. During translation, a conflict might occur between shortcuts that are part of the same active view, resulting in ambiguities between shortcuts.
+
+Below is an example showing a conflict for the `Alt+R` shortcut in the Spanish GUI. Observe that by pressing `Alt`, shortcut hints are displayed under the `<Key>` of each option. **The key `R` is part of three different options, resulting in ambiguity**.
+
+<img width="450" alt="gui shortcut ambiguity" src="https://user-images.githubusercontent.com/18506343/191549555-afcfd791-e21c-47ea-b63e-38b34cee0cdb.gif">
+
+This is a pretty simple test - **navigate in the GUI, use the shortcuts and see if you can find any more ambiguities**.
+
+This kind of issues are responsibility of the translators. If you want to directly help with a fix, see [doc/translation_process.md](https://github.com/bitcoin-core/gui/blob/master/doc/translation_process.md). The best way to report this is to open an issue on [Bitcoin Core's Transifex page](https://www.transifex.com/bitcoin/bitcoin/), see how on [Comments and Issues section on Transifex docs](https://docs.transifex.com/translation/tools-in-the-editor#comments-and-issues). Alternatively, comment [here](https://github.com/bitcoin-core/gui/issues/126) with details.
+
+### Testing the Restore Wallet option
+
+Restoring a wallet from a backup file in the GUI is now possible! Up until now, you could create a backup but the only option was to restore using the RPC interface. This change makes it easier for non-technical users to restore backups.
```

Comment:
if it's someone first foray into using the gui, it might not be apparent how you would go about creating a backup, You should atleast be explicit about the menu options that need to be clicked in order to accomplish this test

Reply:
Good point, I'll add the menu options sequence.